log-collector/Dockerfile.cleanup

```
@@ -31,13 +31,16 @@ RUN --mount=type=cache,target=/tmp/poetry_cache \
 RUN poetry export --format requirements.txt --output constraints.txt --without-hashes
 
-FROM public.ecr.aws/docker/library/python:3.10-alpine AS runtime
+FROM public.ecr.aws/docker/library/python:3.13-alpine AS runtime
+
+RUN apk upgrade --no-cache libssl3 libcrypto3
 
 ENV PYTHONUNBUFFERED=1
 
 COPY --from=build /app/dist/*.whl ./
 COPY --from=build /app/constraints.txt ./
 RUN --mount=type=cache,target=/root/.cache/pip \
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)